lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
+  @moduledoc false
+
+  alias Screens.V2.WidgetInstance.ReconstructedAlert
+
+  defstruct alert: nil
+
+  @type t :: %__MODULE__{
+          alert: String.t()
+        }
+
+  # Temporarily using subway alert type
+  def serialize(%__MODULE__{alert: %Screens.Alerts.Alert{header: header}}) do
+    %{alert_header: header}
+  end
+
+  # Priority will either be 3 if it's a regular alert
+  #     or 1 if a takeover
+
+  # slot_names will be :large if it's a regular alert
+  #     or body_left_takeover / body_right_takeover if a takeover (how can 1 alert fill both sides??)
+
+  defimpl Screens.V2.WidgetInstance do
+    def priority(_instance), do: [3]
```

Comment:
At this moment, the SubwayStatus never appears if the priority of the alerts is [2].  There are too many alerts, so it wants to fill all the space.  Can I keep it at [3] for now, since this will get reworked in the later tickets anyway?

Reply:
Hmm, that's interesting. If that's the case, we can address it later.